Update documentation for vsphere / 4.17 (#398)
appuio/openshift4-docs: docs/modules/ROOT/partials/install/prepare-commodore.adoc

For VSHN, this configuration is set in the https://git.vshn.net/syn/commodore-defaults/-/merge_requests/1789[Commodore global defaults (internal)].
====
+
[NOTE]
====

> If you have a non-standard pod network, you need to ensure to include this in the configuration.

Reviewer: ```suggestion
If you have a non-standard pod network, you need to ensure to adjust the Cilium configuration accordingly:
```
